app/helpers/nhs_number_validator_helper.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module NhsNumberValidatorHelper
+  def validate_nhs_number_correctness(number)
+    puts number.inspect
+    if number.nil?
```

Comment:
I know you've cleaned the number up above, but what about `.blank?` that will validate the empty string or a nil string or something like `"  "` if I remember rightly

Reply:
ah, good shout, and we want the empty string to return too right?